handles.js

```diff
@@ -0,0 +1,73 @@
+/* jshint esversion:6*/
+const handles = function () {
+  return {
+    getHandle : getHandle,
+    postHandle : postHandle,
+    postReset : postReset,
+    putHandle : putHandle,
+    deleteHandle : deleteHandle,
+
+  };
+};
+
+let buzzArr = [];
+let score = 0;
+
+function getHandle (req, res){
+  let buzzwords = buzzArr.map( (obj) => {
+    return obj.buzzword;
+  });
+  let buzzObj = {buzzwords};
+  res.send(buzzObj);
+}
+
+function postHandle (req, res){
+  let matches = req.body.buzzword.match(/\d+/g);
+  if (isNaN(req.body.points) || matches != null){
+    res.json({'success': false});
+  }else{
+    let num = parseInt(req.body.points);
+    req.body.points = num;
+    buzzArr.push(req.body);
+    res.json({'success' : true});
+  }
+}
+
+
+function postReset (req, res){
+  if (req.body.reset === 'true'){
+   buzzArr = [];
+   score = 0;
+   res.json({'success': true});
+ }else{
+  res.json({'success' : false});
+}
+}
+
+function putHandle (req, res){
+  let wordPut = buzzArr.reduce( (obj) => {
+    if (req.body.buzzword === obj);
```

Comment:
So this if statement is a bit irregular and difficult to read. I don't see a particular need for it to be this way either. Also, I believe your comparing an object to a string here. So I don't think this will ever evaluate to true. I believe you're always returning the last item in the array.